docs/components/empty-state.md

```diff
@@ -281,8 +378,99 @@ Slot for providing your custom action button.
 </KEmptyState>
 ```
 
-## Events
+### feature-icon
 
+For each feature provided through the [`features` prop](#features), you can pass a custom icon through the `feature-key-icon` slot where `key` is the value of the `key` property in the corrsponding feature object.
+
+<KEmptyState
+  :features="features"
+  message="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla nec purus feugiat, molestie ipsum et, consequat nibh."
+  title="Empty State With Features"
+>
+  <template #feature-wave-icon>
```

Comment:
I would expect the key to be at the end of the dynamic slot name, not in the middle. 

@Justineo thoughts?

Reply:
I was gonna say let’s stick to our current convention but I found that currently it’s inconsistent. `<KTabs>` has `${key}-anchor` while `<KTableView>` has `tooltip-${key}`…